docs/topics_5/installing-web-console-on-openshift.adoc

```diff
@@ -62,11 +62,22 @@ Two templates are provided, one for shared storage and one without shared storag
 . Click the *{ProductName}* template.
 . Click *Instantiate Template*.
 endif::[]
-ifdef::ocp-45,ocp-41[]
+ifdef::ocp-45[]
 . Review the application settings and click *Create*.
 . In the *Topology* view, wait for the pods to start.
 . Click the *Open URL* button of the `mta` Pod to open the {WebName} in a new browser window.
 endif::[]
+ifdef::ocp-41[]
+. Review the application settings and click *Create*.
+. In the *Topology* view, wait for the pods to start.
+. Click the *Open URL* button of the `mta` Pod to open the {WebName} in a new browser window.
```

Comment:
"in a new browser window over HTTP".
`. Copy the URL and navigate to `\https://secure-mta-web-console-$<{WebName}_URL> to access the {WebName} using HTTPS.`

Are you sure that the whole URL is pasted after the `$`? Does that include the http? It would be good to include an example.

Delete the note.

Same for ocp-311.